packages/xo-web/src/xo-app/logs/backup-ng-logs.js

```diff
@@ -107,10 +102,10 @@ const LOG_COLUMNS = [
   },
 ]
 
-const showTasks = log =>
+const showTasks = (log, { jobs }) =>
   alert(
     <span>
-      {_('jobModalTitle', { job: log.jobId.slice(4, 8) })}{' '}
+      {get(() => jobs[log.jobId].name)}{' '}
```

Comment:
What if we don't know the job?

We should display something anyway.

Reply:
The job's `runId`  is also displayed.